lib/ops/opsservice/deploy.go

```diff
@@ -107,7 +107,11 @@ func remoteDirectories(operation ops.SiteOperation, server *ProvisionedServer, m
 	chownList := []string{
 		server.InGravity("local"),
 		server.InGravity("teleport"),
-		server.InGravity("planet"),
+		server.InGravity("planet", "etcd"),
+		server.InGravity("planet", "log"),
+		server.InGravity("planet", "registry"),
+		server.InGravity("planet", "share"),
+		server.InGravity("planet", "state"),
```

Comment:
I'm not in favor of bulk chowning since we might run into issues with specific directories (i.e. docker-related) as we cannot really tell which parts of the directory should be available for reading/execution for other (non-planet) users/groups. Usually, when not on a SELinux-enabled system and using root (which has `DAC_OVERRIDE`) it is no problem - it starts to fail once SELinux gets at validation.